README.md

```diff
@@ -60,6 +60,24 @@ are assuming terminal/shell commands.
 5. If you get the message `Web application available at http://127.0.0.1:3000` then mojolicious is running and waiting
    for any requests.
 
+### Getting the standalone renderer code running
+
+1. Clone the repository with `git clone --recursive https://github.com/drdrew42/renderer`
+
+2. copy `render_app.conf.dist` to `render_app.conf` and make any desired modifications including changing the ports
+from 3000 to 3001
```

Comment:
Because quasar is proxying the renderer at `/renderer`, the following configurations are necessary in `render_app.conf`:
* SITE_HOST => 'http://localhost:3000' (point at quasar)
* baseURL => '/renderer' (proxy endpoint for SITE_HOST)

Assuming we are playing MITM, users should never have access to the renderer directly.

Reply:
Since the dev server is rewriting /renderer to root, it isn't strictly necessary to set the baseURL setting of the renderer.  Although it would probably be better to do it that way rather than using a rewrite.

Reply:
In terms of actual deployment, this will all need to be done differently in any case.  The webpack dev server is not meant for production use.

Reply:
On `SITE_HOST`, I was using :3001 for this, however I think that was mainly for pulling js libraries when the renderer was sending a full HTML page.  I have stripped it down to only the problem within the `<form>` tag.   

Also, I've kept the `baseURL` as `/` but maybe because again pulling out only the problem.